.github/pull_request_template.md

```diff
@@ -19,6 +19,20 @@
   the functionality of your change
 -->
 
+## Changelog
+<!--
+Add each of the changelog entries as a line item in this section. e.g.
+- fix: Updates the graph
+- feat: Adds a new feature
+
+If this change does not need to added to the changelog, just add a single `skip` line e.g.
+- skip
+
+Valid prefixes: breaking, chore, deprecate, doc, feat, fix, infra, refactor, test
+
+Descriptions following the prefixes must be 50 characters or less
```

Comment:
Whats the reason for this limitation? Is it a technical limitation?

Reply:
Hi @ashwin-pc ,

The prefixes were the ones you suggested in your proposal, and we decided to follow them. Thinking is better to have a fixed categorization than an open-ended one. Regarding the second, we though that a comment for a description of a changeset should be concise, avoiding something really long. If not, a contributor can put a paragraph there which is something we would like to avoid, or not?

We are open to what you guys decide of course.